plugins/bigquery/dbt/include/bigquery/macros/adapters.sql

```diff
@@ -62,18 +77,25 @@
   {%- set raw_kms_key_name = config.get('kms_key_name', none) -%}
   {%- set raw_labels = config.get('labels', []) -%}
   {%- set sql_header = config.get('sql_header', none) -%}
+  
+  {%- set partition_by_dict = adapter.parse_partition_by(raw_partition_by) -%}
+  {%- set is_scripting = (temporary == 'scripting') -%} {# "true" temp tables only possible when scripting #}
```

Comment:
Can you just remind me where we ended up on this? BigQuery temp tables can't be used outside of scripting?